### DISCUSSION TITLE
mmrm 0.3.3 to CRAN

### DISCUSSION THREAD

**Discussion by mrmorgan17** [openpharma/mmrm#356] [General]
I was just wondering when mmrm version 0.3.3 will be pushed to CRAN. It seems close and I know I can use the development version in the meantime but if anyone has a good estimate when the new version will be pushed to CRAN that would be much appreciated

**Comment by danielinteractive** [comment]
Yeah it is hanging at the CRAN team right now so ETA is tricky :)

**Reply by mrmorgan17** [reply]
Good to know, this is such a well developed and documented package. It's worth the wait

**Reply by danielinteractive** [reply]
Almost there ... we should be getting 0.3.5 on CRAN very soon

**Reply by danielinteractive** [reply]
still waiting ... sorry that it takes so long...

**Reply by danielinteractive** [reply]
now submitted 0.3.6.... keeping fingers crossed

**Comment by danielinteractive** [comment]
Finally!
